samples/word/40-tables/manage-formatting.yaml

```diff
@@ -0,0 +1,281 @@
+order: 2
+id: word-tables-manage-formatting
+name: Get the formatting of a table
+description: 'Gets the formatting details of a table, a table row, and a table cell, including borders, alignment, and cell padding.'
+author: OfficeDev
+host: WORD
+api_set:
+    WordApi: '1.3'
+script:
+    content: |
+        $("#get-table-alignment").click(() => tryCatch(getTableAlignment));
+        $("#get-table-border").click(() => tryCatch(getTableBorder));
+        $("#get-table-cell-padding").click(() => tryCatch(getTableCellPadding));
+        $("#get-table-row-alignment").click(() => tryCatch(getTableRowAlignment));
+        $("#get-table-row-border").click(() => tryCatch(getTableRowBorder));
+        $("#get-table-row-cell-padding").click(() => tryCatch(getTableRowCellPadding));
+        $("#get-table-cell-alignment").click(() => tryCatch(getTableCellAlignment));
+        $("#get-table-cell-border").click(() => tryCatch(getTableCellBorder));
+        $("#get-table-cell-cell-padding").click(() => tryCatch(getTableCellCellPadding));
+        $("#setup").click(() => tryCatch(insertTable));
+
+        async function getTableAlignment() {
+          // Get alignment details about the first table in the document.
```

Comment:
When you say "alignment details" here, is it in regards to the table itself, of the contents of the table? I would make similar clarifications for the other three alignment descriptions.

Reply:
For table, it's a combination - I explain with the logs. I did clarify the alignment description for the others